SmallBetterLogger/SmallBetterLogger.hpp

```diff
@@ -0,0 +1,179 @@
+#ifndef SMALL_BETTER_LOGGER_H
+#define SMALL_BETTER_LOGGER_H
+
+// Used for writing to output stream
+#include <iostream>
+#include <fstream>
+
+// Used for formatting and creating the output string
+#include <sstream>
+#include <vector>
+#include <regex>
+
+namespace sbl
```

Comment:
I know having a lot of shorts is somehow customary to cpp, but I feel like this namespace should be a little more suggestive. Either a full name SmallBetterLogger, or at least SBLogger to imply the usage and content.